src/taco-cli/resources/en/resources.json

```diff
@@ -425,6 +425,9 @@
     "RemoteBuildUnsupportedPlatform": "Error: Platform '{0}' not supported by current version of Cordova.",
     "_RemoteBuildUnsupportedPlatformcomment": "Error message printed when trying to 'taco remote add <platform>' and the platform is not supported. {0} is the platform",
 
+    "InvalidCordovaWithNode5": "Incompatible configuration: Unable to use Cordova < 5.4.0 with Node >= 5.0.0. Either upgrade to a newer version of Cordova, or revert to an older version of node. See http://go.microsoft.com/fwlink/?LinkID=618471 for more details.",
```

Comment:
This is being addressed and should be updated in a couple of days; prior to release at least.